url.go

```diff
@@ -22,29 +22,23 @@ type Location interface {
 	String() string
 }
 
-// URL represents a fully resolved charm location with a specific series, such
-// as:
+// URL represents a charm location:
 //
 //     cs:~joe/oneiric/wordpress
 //     cs:oneiric/wordpress-42
 //     local:oneiric/wordpress
+//     cs:~joe/wordpress
+//     cs:wordpress
+//     cs:precise/wordpress-20
```

Comment:
Oh, I see this convention is also mentioned in godocs below. I hadn't realized we'd already added support for this. Cool.